### PR TITLE
✨Add Miscellaneous additions

### PR DESCRIPTION
**Pull request by Shazuub**

### CODE REVIEW

**Review by MoskalykA — CHANGES_REQUESTED** [review]
You took the old file from API_Player:showKeyboard

File: src/api/player/client/show3DText.lua [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---show3dtext
+---@param x number
+---@param y number
+---@param z number
+---@param text string
```
Comment (PABLO-1610) [inline src/api/player/client/show3DText.lua:5]:
```suggestion
---@param text string
---@return void
---@public
```

File: src/api/player/client/showKeyboard.lua [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
+        return result
     else
-        Citizen.Wait(500)
-        return (nil)
+        Wait(500)
+        return nil
     end
-end
\ No newline at end of file
+end
```
Comment (PABLO-1610) [inline src/api/player/client/showKeyboard.lua]: ???????????????????

File: src/api/player/client/showSubtitle.lua [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---subtitle
+---@param text string
```
Comment (PABLO-1610) [inline src/api/player/client/showSubtitle.lua:2]:
```suggestion
---subtitle
---@param text string
---@return void
---@public
```

File: src/api/props/server/createProps.lua [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---createprops
+---@param props string
+---@param X number
+---@param Y number
+---@param Z number
+---@param H number
```
Comment (PABLO-1610) [inline src/api/props/server/createProps.lua:6]:
```suggestion
---@param H number
---@return void
---@public
```

File: src/api/props/server/createProps.lua [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+function API_Props:createprops(props,X,Y,Z,H)
+    local object = CreateObject(props, X, Y, Z - 1, true)
```
Comment (PABLO-1610) [inline src/api/props/server/createProps.lua:8]:
```suggestion
function API_Props:createprops(props,x,y,z,h)
    local object = CreateObject(props, x, y, z - 1, true)
```

File: src/api/props/server/createProps.lua [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+    SetEntityHeading(object, H)
```
Comment (PABLO-1610) [inline src/api/props/server/createProps.lua:9]:
```suggestion
    SetEntityHeading(object, h)
```

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]

**Review by PABLO-1610 — APPROVED** [review]

**Pull request merged**